docs/troubleshoot/verify-fingerprint.md

```diff
@@ -32,14 +39,11 @@ When you transfer the PAX file between systems, you must use binary transfer mod
    - `RefRuntimeHash-1.13.0.txt` (text) 
    - `zowe-verify-authenticity.sh` (text)
 
-Each `RefRuntimeHash-v.r.m.txt` file is specific to a Zowe release, where `v.r.m` is your Zowe release number, e.g. `1.9.0`.  Further 
-`RefRuntimeHash-v.r.m.txt` files will be added to the PAX file as each release becomes available.  This includes releases after v1.14.  
-
+Each `RefRuntimeHash-V.v.p.txt` file is specific to a Zowe release, where `V.v.p` is your Zowe release number, for example, `1.9.0`. This list of files is updated to include new Zowe releases as they become available. For example, if you use Zowe version 1.14, you will see `RefRuntimeHash-1.14.0.txt` in the list. 
 
+## Step 2: Verify your runtime directory
 
-## Step 2: Verify your runtime folder
-
-Now you are ready to verify your `ROOT_DIR` runtime folder, for example, `/usr/lpp/zowe/v1.14`, which contains the following files, which you can show by using the `ls` command. 
+Now you are ready to verify your runtime directory `ROOT_DIR`, for example, `/usr/lpp/zowe/v1.14`, which contains the following files. You can show these files by using the `ls` command. 
```

Comment:
the list of filenames on line 50  wraps untidily
```
bin components fingerprint install_log manifest.json
scripts
```

Reply:
done